nautobot/extras/models/jobs.py

```diff
@@ -472,30 +472,26 @@ class JobResult(BaseModel, CustomFieldModel):
         help_text="Current state of the Job being run",
         db_index=True,
     )
-    data = models.JSONField(encoder=DjangoJSONEncoder, null=True, blank=True)
+    result = models.JSONField(
```

Comment:
Do we have any tests that confirm what happens if a job returns something that isn't JSON-serializable?